contracts/lib/LibLiquidation.sol

```diff
@@ -97,7 +97,7 @@ library LibLiquidation {
         int256 maxSlippage,
         int256 avgPrice,
         LibLiquidation.LiquidationReceipt memory receipt
-    ) internal pure returns (uint256) {
+    ) public pure returns (uint256) {
```

Comment:
Are these only public purely for testing purposes? Wondering if there is an easier way to test libs than jsut having to make the functions public

Reply:
I couldn't find any way. We were talking about it in slack yesterday and figured it was fine. Also there's no risk of them being public since they are pure